pkg/util/ovs/ovs.go

```diff
@@ -5,9 +5,19 @@ import (
 	"fmt"
 	"strings"
 
-	"github.com/openshift/openshift-sdn/pkg/exec"
+	"k8s.io/kubernetes/pkg/util/exec"
 )
 
+var execer exec.Interface
```

Comment:
We could do it like kubernetes does, and make ovs.NewTransaction() take an exec.Interface argument, and then real callers would do `otx := ovs.NewTransaction(kexec.New(), "br0")`, and the test code would do `otx := ovs.NewTransaction(&fakeExec, "br0")`